src/site/_filters/i18n.js

```diff
@@ -64,9 +64,10 @@ const data = {i18n: walk(path.join(__dirname, '..', '_data', 'i18n'))};
  * Looks for the i18n string that matches the path and locale.
  * @param {string} path A dot separated path
  * @param {string} [locale] A locale prefix (example: 'en', 'pl')
+ * @param {string} fallback A fallback string for no i18n result
  * @return {string}
  */
-const i18n = (path, locale = defaultLocale) => {
+const i18n = (path, locale = defaultLocale, fallback = null) => {
```

Comment:
What's that needed for?

A problem with this is also that webdev-infra now contains i18n code which is meant to replace the one here.

Reply:
I added the fallback for no i18n result because I added this `bio: i18n.authors.${key}.bio` to read the translation for the bio key in each author, see [this file](https://github.com/GoogleChrome/web.dev/pull/9872/files#diff-90d12317ec6c6ca3f3aaf045b4132be3e189606ccd6fc5783fab51fcc937ca86).

The current logic is if there is no i18n result, it will throw the error `Could not find i18n result for: ${path}`, and each author in this author.yaml has no bio key yet. So according to the current logic, Is that okay if I return an empty string instead of throwing the error message in the case of no i18n result?